scripts/monitoring/cron-send-create-app.py

```diff
@@ -297,31 +109,56 @@ def getPodStatus(pod):
 
     return "%s %s" % (pod['metadata']['name'], pod['status']['phase'])
 
-def setup(config, oocmd=None,):
+def getPod(name):
+    """ get Pod from all possible pods """
+    pods = ocutil.get_pods()
+    for pod in pods['items']:
+        if pod and pod['metadata']['name'] and pod['metadata']['name'].startswith(name):
+            # only report on running build pods, we'll wait if build pod is pending
+            if pod['metadata']['name'].endswith("build") and pod['status']['phase'] != 'Running':
+                continue
+            else:
+                return pod
+
+def setup(config):
     """ global setup for tests """
     logger.info('setup()')
-    oocmd.delete_project()
-    time.sleep(5)
-    oocmd.new_project()
-    time.sleep(5)
+    logger.debug(config)
 
-    return config
+    project = None
+
+    try:
+        project = runOCcmd_yaml("get project {}".format(config.namespace))
+        logger.debug(project)
+    except Exception:
+        pass # don't want exception if project not found
+
+    if not project:
+        try:
+            runOCcmd("new-project {}".format(config.namespace))
+            time.sleep(commandDelay)
+        except Exception:
```

Comment:
This has been tested in preview, we can push oadm command through this function if required, but it does work :-)